UtilityFunctions/LukeMeilerHW7P2.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<p>That's an interesting filter ....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_2DFFT/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>